cmd/kpod/main.go

```diff
@@ -54,12 +63,9 @@ func main() {
 			Name:  "storage-opt",
 			Usage: "used to pass an option to the storage driver",
 		},
-		cli.StringFlag{
-			Name:  "config, c",
-			Usage: "path of a config file detailing container server configuration options",
-		},
 	}
 	if err := app.Run(os.Args); err != nil {
-		logrus.Fatal(err)
+		logrus.Errorf(err.Error())
```

Comment:
logrus.Fatal () Dumps the stack, making for a simple
container not found 

Look like a segfault.